pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -75,6 +73,10 @@ type Reconciler struct {
 // Check that our Reconciler implements controller.Reconciler
 var _ controller.Reconciler = (*Reconciler)(nil)
 
+type env struct {
```

Comment:
nit: can you call this struct `envConfig` as we are doing in other places? For example: [here](https://github.com/knative/eventing/blob/133c7cca04baf7545c87cc8d504d298b93535141/cmd/cronjob_receive_adapter/main.go#L32)

Reply:
Done.